autogpt_platform/docker-compose.platform.yml

```diff
@@ -36,6 +36,15 @@ services:
       interval: 10s
       timeout: 5s
       retries: 5
+  rabbitmq:
```

Comment:
Looks like you've missed out adding this as a dependency of relevant images

Reply:
correct